js/mixins/reduce-array.js

```diff
@@ -11,15 +11,42 @@ import '../parts/Utilities.js';
 var reduce = H.reduce;
 
 var reduceArrayMixin = {
+    /**
+     * Get min value of array filled by OHLC data.
+     * @param {array} arr Array of OHLC points (arrays).
+     * @param {string} index Index of "low" value in point array.
+     * @returns {number} Returns min value.
+     */
     minInArray: function (arr, index) {
         return reduce(arr, function (min, target) {
             return Math.min(min, target[index]);
         }, Infinity);
     },
+    /**
+     * Get max value of array filled by OHLC data.
+     * @param {array} arr Array of OHLC points (arrays).
+     * @param {string} index Index of "high" value in point array.
+     * @returns {number} Returns max value.
+     */
     maxInArray: function (arr, index) {
-        return H.reduce(arr, function (min, target) {
+        return reduce(arr, function (min, target) {
             return Math.max(min, target[index]);
         }, 0);
+    },
+    /**
+     * Get extremes of array filled by OHLC data.
+     * @param {array} arr Array of OHLC points (arrays).
+     * @param {string} minIndex Index of "low" value in point array.
+     * @param {string} maxIndex Index of "high" value in point array.
+     * @returns {array} Returns array with min and max value.
+     */
+    getArrayExtremes: function (arr, minIndex, maxIndex) {
+        return reduce(arr, function (prev, target) {
+            return [
+                Math.min(prev[0], target[minIndex]),
+                Math.max(prev[1], target[maxIndex])
+            ];
+        }, [Infinity, 0]);
```

Comment:
Should be `Number.MAX_VALUE` and `-Number.MAX_VALUE`.

Reply:
Fixed.